docs/ref/models/expressions.txt

```diff
@@ -640,3 +640,28 @@ Let's see how it works::
     Apple: AAPL
     Yahoo: Internet Company
     Django Software Foundation: No Tagline
+
+Adding support for third party backends
+---------------------------------------
+If you are using a backend that uses a different syntax for a certain function,
+you can easily add support for it by monkey patching a new method onto the
+relevant expression class.
+
+Let's say we are using the `Firebird` backend. If `Firebird` would use a
```

Comment:
Ah, sorry! I wasn't aware of that recommendation/requirement. Ignore this comment.